great_expectations/data_context/data_context.py

```diff
@@ -3264,6 +3264,38 @@ def run_checkpoint(
             **kwargs,
         )
 
+    def add_profiler(
+        self,
+        name: str,
+        config_version: float,
+        rules: Dict[str, dict],
+        class_name: str,
+        module_name: Optional[str] = None,
+        variables: Optional[dict] = None,
+        commented_map: Optional[CommentedMap] = None,
```

Comment:
is there a config object we could use here instead, or a business object? What's the use-case for commented map?

Reply:
I don't believe commented_map is actually necessary here - we require it when reading from YAML but this method shouldn't need it.